scripts/apps/master-desk/components/OverviewComponent.tsx

```diff
@@ -56,7 +56,9 @@ export class OverviewComponent extends React.Component<IProps, IState> {
             },
             {
                 archive: {
+                    create: true,
                     update: [
+                        'state',
```

Comment:
It'd be better fields didn't have to be hardcoded, but would instead listen for changes on query fields, like in `with-live-query.tsx` for example.

Reply:
here it's doing aggregations and none of those fields are actually returned, not sure how to best specify that

Reply:
It'd be good to have this list of fields in a single place.

If we are sending fields on which to aggregate from client to server, then we could use the same fields(in client code) to decide whether to refresh data.

If we don't send the fields by which to aggregate and server and it is hardcoded on the server, it would be better to use a custom websocket event that would only fire when aggregated fields change.

Reply:
will give it a try and might introduce it in another PRs